src/mmw/js/src/modeling/views.js

```diff
@@ -324,6 +328,52 @@ var ScenarioButtonsView = Marionette.ItemView.extend({
             compareUrl: compareUrl
         };
     },
+
+    showCompare: function() {
+        var scenarios = this.projectModel.get('scenarios'),
+            runoffTable = [
+                {
+                    name: "Runoff",
+                    unit: "cm",
+                    values: scenarios.map(function(s) {
+                        return s.get('results')
+                                .findWhere({ name: "runoff" })
+                                .get('result')
+                                .runoff.modified.runoff;
```

Comment:
This is pretty brittle, and won't allow us to show the compare view when the model is polling or if one of the scenarios has an error. Want me to open an issue to deal with this, or do you want to handle it now?